css/files.css

```diff
@@ -1,3 +1,30 @@
+div.files-list__header:has(div.files-list__breadcrumbs) {
```

Comment:
todo: the breadcrumb in the target dialog is not yet customized

Reply:
My suggestion to make it work for target dialog:

```
div.files-list__header:has(div.files-list__breadcrumbs), .file-picker {
	div.files-list__breadcrumbs, .file-picker__breadcrumbs {
		ul.breadcrumb__crumbs {
			a, button:not(.button-vue--icon-only) {

			}
		}
	}
}
```